private/rules/pom_file.bzl

```diff
@@ -26,6 +26,9 @@ def _explode_coordinates(coords):
     if len(parts) == 4:
         # Assume a buildr coordinate: groupId:artifactId:type:version
         return (parts[0], parts[1], parts[3], parts[2])
+    if len(parts) == 5:
+        # Assume groupId:artifactId:type:classifier:version
```

Comment:
nit:  maven refers to what you have as `type` as `packaging` https://maven.apache.org/pom.html#packaging